### ISSUE TITLE
Question on fine-tuning 

### ISSUE THREAD

**Issue by wangzixing1998** [vitoralbiero/img2pose#67]
great job.  now I'm focusing on fine-tuning.  Now I make LMDB file by 300wlp annotation you provided.  As mentioned in your paper, 
“Training pose rotation labels are obtained by converting the 300W-LP ground-truth Euler angles to rotation vectors, and pose translation labels are created using the ground-truth landmarks, using standard means”
I get the rotation vector in the Json_loader.py:

`            for item in range(len(image_json["pose_para"])):`
                `  euler = image_json["pose_para"][item]`
                `  r = Rotation.from_euler("xyz",euler, degrees=False)`
                `  x = r.as_euler('xyz', degrees=True)`
                `  x[1] = -x[1]`
                `  x[2] = -x[2]`
                `  r_2 = Rotation.from_euler("xyz",x, degrees=True)`
                `  rot_mat = np.transpose(r_2.as_matrix())`
                `  rotvecs.append(Rotation.from_matrix(rot_mat).as_rotvec())`
 and use the function of "get_pose()" to get the translation vector.  
However, in the process of model training, the total loss is as high as 100, and the accuracy of the estimated head pose of the fine-tuned model is lower than that before fine-tuning.

I don't know which step of the code is wrong, can you provide a detailed code for fine-tuning ？

**Comment by vitoralbiero** [comment]
Hello @wangzixing1998,

Thanks for your interest in our work!

When creating the lmdb file, try the JSON loader that I have just added [here](https://github.com/vitoralbiero/img2pose/blob/main/utils/json_loader_300wlp.py).
Then, when fine-tuning, disable augmentations, as this dataset is already augmented.

Hope this helps.
